android/src/main/java/com/github/reactNativeMPAndroidChart/charts/YAxisChartBase.java

```diff
@@ -21,10 +21,10 @@
 
     protected void setYAxisConfig(YAxis axis, ReadableMap propMap) {
         if (BridgeUtils.validate(propMap, ReadableType.Number, "axisMaxValue")) {
-            axis.setAxisMaxValue((float) propMap.getDouble("axisMaxValue"));
+            axis.setAxisMaximum((float) propMap.getDouble("axisMaxValue"));
```

Comment:
Update prop name to match new method name. `axisMaxValue` -> `axisMaximum`.
Please do that for all method changes.

Additionally we could keep old names alongs side new and add deprecation warning if old is used.